snmp/datadog_checks/snmp/utils.py

```diff
@@ -74,6 +84,24 @@ def recursively_expand_base_profiles(definition):
         definition.setdefault('metric_tags', []).extend(base_definition.get('metric_tags', []))
 
 
+def get_default_profiles():
+    """Return all the profiles installed on the system."""
+    # type: () -> Dict[str, Any]
+    profiles = {}
+    paths = [_get_profiles_site_root(), _get_profiles_confd_root()]
+    for path in paths:
+        if not os.path.isdir(path):
+            continue
+        for filename in os.listdir(path):
+            if filename.startswith('_'):
+                continue
+            base, ext = os.path.splitext(filename)
+            if ext != '.yaml':
+                continue
+            profiles[base] = {'definition_file': os.path.join(path, filename)}
+    return profiles
```

Comment:
(Nit) I think we should encourage always giving code some "air" in the SNMP integration… So, maybe like this…? 😅

```suggestion
    profiles = {}
    paths = [_get_profiles_site_root(), _get_profiles_confd_root()]

    for path in paths:
        if not os.path.isdir(path):
            continue

        for filename in os.listdir(path):
            if filename.startswith('_'):
                continue

            base, ext = os.path.splitext(filename)
            if ext != '.yaml':
                continue

            profiles[base] = {'definition_file': os.path.join(path, filename)}

    return profiles
```